scripts/build_linux.sh

```diff
@@ -20,7 +22,8 @@ fortran)
   make && make run
   ;;
 dpc++)
-  source /opt/intel/oneapi/tbb/2021.1-beta10/env/vars.sh
+# shellcheck source=/dev/null
```

Comment:
Shellcheck flags sourcing of non-constant paths. This comment disables this check for the next line.